ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/PaywallListener.kt

```diff
@@ -10,6 +10,7 @@ interface PaywallListener {
     fun onPurchaseStarted(rcPackage: Package) {}
     fun onPurchaseCompleted(customerInfo: CustomerInfo, storeTransaction: StoreTransaction) {}
     fun onPurchaseError(error: PurchasesError) {}
+    fun onPurchaseCancelled() {}
```

Comment:
This is a breaking change, but it's marked as experimental, so it's probably fine to release as minor

Reply:
Actually, it has a default implementation so we are good!

Reply:
Talked about this offline. This has a default implementation so it's optional for developers.

Reply:
Hmm while this is an easier API, I wonder if it might be confusing since `PurchasesError` also has the `PurchaseCancelledError` code, which will be unused now...

So this could be considered a breaking change for people that have been listening to that instead.

Reply:
That's fair, but luckily Android paywalls is still marked as experimental so we could fix this on time 😇